apps/designer/app/routes/designer/$id.tsx

```diff
@@ -35,17 +38,42 @@ type Error = {
 
 export const action: ActionFunction = async ({ request, params }) => {
   if (params.id === undefined) throw new Error("Project id undefined");
+  try {
+    if (request.method === "DELETE") {
+      const formData = await request.formData();
+      if (formData.get("assetId")) {
+        const id = formData.get("assetId") as string;
+        const name = formData.get("assetName") as string;
```

Comment:
instead of asserting with typescript, validate mit zod

Reply:
Will do

Reply:
Mit zot, went all German there ahahah

Reply:
Done with zod-form-data, updated other places to use it